김진/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,18 @@
+function solution(s) {
+  upper_s = s.toUpperCase();
+  let pNum = 0;
+  let yNum = 0;
+  for (let i = 0; i < upper_s.length; i++) {
+    if (upper_s[i] == "P") {
+      pNum++;
+    } else if (upper_s[i] == "Y") {
+      yNum++;
+    }
+  }
+
+  if (pNum == yNum || (pNum == 0 && yNum == 0)) {
```

Comment:
```suggestion
    if (upper_s[i] === "P") {
      pNum++;
    } else if (upper_s[i] === "Y") {
      yNum++;
    }
  }

  if (pNum === yNum) {
```

특별한 경우 아니면 ===을 추천합니다. 그리고 굳이 논리 연산자를 사용하지 않아도 될 것 같아요.